Makefile

```diff
@@ -195,19 +195,37 @@ docker-buildx-dev: test
 	@if ! DOCKER_CLI_EXPERIMENTAL=enabled docker buildx ls | grep -q container-builder; then\
 		DOCKER_CLI_EXPERIMENTAL=enabled docker buildx create --name container-builder --use;\
 	fi
-	DOCKER_CLI_EXPERIMENTAL=enabled docker buildx build --platform "linux/amd64,linux/arm64,linux/arm/v7" \
-		-t $(REPOSITORY):$(DEV_TAG) \
-		-t $(REPOSITORY):dev \
+	DOCKER_CLI_EXPERIMENTAL=enabled docker buildx build --platform "linux/amd64" \
```

Comment:
This isn't the issue and also defeats the purpose of a multi-arch image where we don't want to re-tag them rather we want an image that is already multi-arch.

This should remain being

```
	DOCKER_CLI_EXPERIMENTAL=enabled docker buildx build --platform "linux/amd64,linux/arm64,linux/arm/v7" \
		-t $(REPOSITORY):$(DEV_TAG) \
		. --push
```

Reply:
This is a workaround for the Quay issue. We can not push same tag to Quay twice using buildx, which will be required for release (pushing latest + version).